src/js/actions/search/stock.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2016 Adobe Systems Incorporated. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+define(function (require, exports) {
+    "use strict";
+
+    var Immutable = require("immutable");
+
+    var events = require("js/events"),
+        locks = require("js/locks"),
+        adapter = require("adapter"),
+        search = require("js/stores/search"),
+        nls = require("js/util/nls");
+
+    var STOCK_URL = search.SEARCH_STOCK_URL;
+    
+    /**
+     * Executes Adobe Stock search by opening a URL with the search term
+     *
+     * @return {Promise}
+     */
+    var _confirmStockSearch = function (searchTerm) {
+        var searchURL = STOCK_URL + searchTerm;
+        return adapter.openURLInDefaultBrowser(searchURL);
+    };
+    _confirmStockSearch.action = {
```

Comment:
It doesn't look like this is an action; it's just a plain function. So, this property isn't needed.